docs/name.com.md

```diff
@@ -0,0 +1,45 @@
+# Name.com
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "name.com",
+      "domain": "domain.com",
+      "host": "@",
+      "username": "username",
+      "password": "password",
+      "ttl": 300
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"`
+- `"host"` is your host and can be a subdomain, `"@"` or `"*"` generally
+- `"username"`
+- `"password"`
+
+### Optional parameters
+
+- `"ttl"` is the time this record can be cached for in seconds. Name.com allows a minimum TTL of 300, or 5 minutes. Name.com defaults to 300 if not provided.
+- `"ip_version"` can be `ipv4` (A records) or `ipv6` (AAAA records), defaults to `ipv4 or ipv6`
+
+## Domain setup
+
+<a href="https://www.name.com"><img src="../readme/name.svg" alt="drawing" width="25%"/></a>
```

Comment:
Can you move the svg file to the docs directory? Otherwise we could end up with a lot svgs at the root directory

Reply:
Never mind, silly me I didn't realize it was in the readme directory.